airbyte-integrations/connectors/source-salesforce/unit_tests/integration/test_bulk_stream.py

```diff
@@ -55,7 +55,34 @@ def _calculate_start_time(start_time: datetime) -> datetime:
     return start_time.replace(microsecond=0)
 
 
-@freezegun.freeze_time(_NOW.isoformat())
```

Comment:
I had to more this annotation to each test specifically because the test that needs the timeout needs time to change so we can't freeze it